readme.md

```diff
@@ -82,20 +84,36 @@ require('@zeit/ncc')('/path/to/input', {
   v8cache: false, // default
   quiet: false, // default
   debugLog = false // default
-}).then(({ code, map, assets }) => {
-  console.log(code);
-  // Assets is an object of asset file names to { source, permissions, symlinks }
-  // expected relative to the output code (if any)
+}).then(({ files, symlinks }) => {
+  // files: an object of file names to { source, permissions }
+  // symlinks: an object of symlink mappings required for the build
```

Comment:
```suggestion
  // symlinks: an array of symlink paths required for the build
```

I think this is an array, right?

Reply:
It's an object of mappings `{ 'local/path': './mapped-path' }`.